src/main/java/dinkplugin/message/NotificationBody.java

```diff
@@ -1,39 +1,40 @@
 package dinkplugin.message;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
-import lombok.Value;
+import lombok.Singular;
 
-import java.util.LinkedList;
 import java.util.List;
 
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
 public class NotificationBody<T> {
-    private NotificationType type;
-    private T extra;
-    private String content;
-    private String playerName;
-
-    @Builder.Default
-    private List<Embed> embeds = new LinkedList<>();
 
+    /*
+     * Dink fields
+     */
     @Builder.Default
     @EqualsAndHashCode.Exclude
     private transient String screenshotFile = "image.png";
+    private NotificationType type;
+    private String playerName;
+    private T extra;
+
+    @SerializedName("text") // used by slack; avoids duplication in discord webhook
+    private String content;
```

Comment:
```suggestion
    private transient String content;
```
Alternatively, we could exclude this text from the json altogether (via `transient`) - I'm open to opinions

Reply:
Unless we have a specific request for this, I don't think we should include it